site/pages/gcweb-theme/implementing.hbs

```diff
@@ -0,0 +1,243 @@
+	<li>Complete a content cleanup exercise that consist at least to:
+		<ul>
+			<li>Taking action for redundant outdated trivial (ROT) content</li>
+			<li>Fixing any HTML lint issue that could be reported by the W3C HTML5 validation software</li>
+			<li>Removing any custom and inline CSS you have. If you can't then at least identify all those pages and  centralize the CSS in one file</li>
+		</ul>
+	</li>
+</ul>
+
+<p>Converting your content pages into GCWeb compatible markup means:</p>
+<ul>
+	<li>You are using valid HTML5 markup</li>
+	<li>There is no custom CSS and no inline CSS</li>
+	<li>You are leveraging the <a href="https://getbootstrap.com/docs/3.3/css/">Bootstrap 3 CSS grid</a></li>
+	<li>You are using GCWeb CSS and WET 4 CSS, documented in:
+		<ul>
+			<li><a href="https://wet-boew.github.io/wet-boew-styleguide/index-en.html">wet-boew style guide</a></li>
+			<li><a href="https://wet-boew.github.io/themes-dist/GCWeb/index-en.html#notes">Notes and other resource in GCWeb</a></li>
+			<li>Following the code sample and <a href="https://wet-boew.github.io/themes-dist/GCWeb/index-en.html#templates-common-design">published page template in GCWeb</a>. Like Topic template.</li>
+		</ul>
+	</li>
+	<li>You are using WET4 plugin and GCWeb plugin
+		<ul>
+			<li><a href="https://wet-boew.github.io/v4.0-ci/demos/index-en.html">WET 4 plugins</a></li>
+			<li><a href="https://wet-boew.github.io/themes-dist/GCWeb/index-en.html#plugins">GCWeb plugins</a></li>
+		</ul>
+	</li>
+</ul>
+
+<p>Note regarding site migrating from WET 3, here some well know chalenges:</p>
+<ul>
+	<li>The grid system, <code>.span-*</code> need to be converted into a combination of <code>div.row</code> and <code>div.col-*-*</code>.</li>
+	<li>Any callout module would need to be converted into alerts or panels in combinaison with a CSS grid. They need to be coded in distinct markup</li>
+	<li>Various of other changes that are not fully documented, like the CSS class mapping</li>
+</ul>
+
+<h2>Stay up to date</h2>
+
+<p>Every release of GCWeb now follow a strict versionning numbering and the release notes now contains all the information your need to update your implementation. That include information like:</p>
+<ul>
+	<li>List of all changes that are marked as Major, Minor or Patch</li>
+	<li>An overview of all changes that was applied to the WET 4 dependancy</li>
+	<li>List of modified files that you would need to replace</li>
+	<li>Link to all Subresource integrity (SRI) hash</li>
+	<li>Adhoc details on specific content update that don't have their own documentation page yet.</li>
+</ul>
+
+<p>Since February 2019, we are using and releasing version in accordance with the semantic versionning 2.0.0 (<a href="https://wet-boew.github.io/wet-boew-documentation/decision/6.html">Design decision #6</a>) where our API is defined by the <a href="https://wet-boew.github.io/wet-boew-documentation/decision/3.html">Design decision #3</a></p>
```

Comment:
```suggestion
<p>Since February 2019, we are using and releasing versions in accordance with the semantic versioning 2.0.0 (<a href="https://wet-boew.github.io/wet-boew-documentation/decision/6.html">Design decision #6</a>) where our API is defined by the <a href="https://wet-boew.github.io/wet-boew-documentation/decision/3.html">Design decision #3</a></p>
```